src/test/resources/application-test.yml

```diff
@@ -2,7 +2,7 @@
 spring:
   data:
     mongodb:
-      port: 27017
+      port: ${mongodb.container.port}
```

Comment:
I don't see the point of using a property placeholder, when you could use the default placeholder `spring.data.mongodb.port`

Reply:
This is for `testcontainers`. It uses a random port in tests. We need to capture the random port in Spring. I will see if I can directly set spring variable in testcontainer.